packages/patternfly-4/react-charts/src/components/ChartBar/examples/ChartBar.md

```diff
@@ -10,14 +10,15 @@ import './chart-bar.scss';
 
 Note: PatternFly React charts live in its own package at [@patternfly/react-charts](https://www.npmjs.com/package/@patternfly/react-charts)!
 
+## Introduction
```

Comment:
Why does this chart MD file have an introduction tag, but not others? Most chart examples have a similar paragraph.

Reply:
We should make them consistent. In our refactor we weren't very consistent.